staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -546,6 +546,11 @@ const (
 	JobReasonBackoffLimitExceeded string = "BackoffLimitExceeded"
 	// JobReasponDeadlineExceeded means job duration is past ActiveDeadline
 	JobReasonDeadlineExceeded string = "DeadlineExceeded"
+	// MaxFailedIndexesExceeded indicates that an indexed of a job failed
+	// https://kep.k8s.io/3850
+	JobReasonMaxFailedIndexesExceeded string = "MaxFailedIndexesExceeded"
+	// FailedIndexes means Job has failed indexes.
+	JobReasonFailedIndexes string = "FailedIndexes"
```

Comment:
```suggestion
	// JobReasonMaxFailedIndexesExceeded indicates that an indexed of a job failed
	// https://kep.k8s.io/3850
	JobReasonMaxFailedIndexesExceeded string = "MaxFailedIndexesExceeded"
	// JobReasonFailedIndexes means Job has failed indexes.
	JobReasonFailedIndexes string = "FailedIndexes"
```

In the above vars, var names seem to be used. Which is correct? var name vs value?

Reply:
Yeah. consistency seems more important here. But overall I prefer to use values, rather than variable names.

Reply:
> But overall I prefer to use values, rather than variable names.

I think so too.